### ISSUE TITLE
" Cannot read property 'replace' of null" at remote: true

### ISSUE THREAD

**Issue by no1-knows** [DavyJonesLocker/client_side_validations-simple_form#75]
### Steps to reproduce*

make simple scaffold one
add remote: true to form tag
change config/initializers/simple_form.rb
commentout ```#b.use :error, wrap_with: { tag: :span, class: :error }```
uncomment: ```b.use :full_error, wrap_with: { tag: :span, class: :error }```
input value at form
submit

### Expected behavior*
save/update value

### Actual behavior*
Uncaught TypeError: Cannot read property 'replace' of null
at simple-form.esm.js:22 in Chrome console

### System configuration*
**Rails version**:
6.0.2.1
**Ruby version**:
2.6.3
**Client Side Validations version**:
16.1.0
**Client Side Validations Simple Form version**:
9.2.0
**Bootstrap version (if used)**:

### Code snippet from your model of the validations*
```validates :name, presence: true, length: { minimum: 5 }```
### The whole form code from your template*
```
<%= simple_form_for @test, validate: true, remote: true  do |f| %>
  <%= f.error_notification %>
  <%= f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present? %>

  <div class="form-inputs">
    <%= f.input :name %>
  </div>

  <div class="form-actions">
    <%= f.button :submit %>
  </div>
<% end %>
```
### The resulting HTML*
```
 
<!DOCTYPE html>
<html>
  <head>
    <title>ClientSideValidationWithSimpleform</title>
    <meta name="csrf-param" content="authenticity_token" />
    <meta name="csrf-token" content="VQ9NH7Iw3wkfAnQkonWiZisJZ1XpGCsoejaFnM9wNj6lEX7V6djen2mtAJNzxsI56wf8vmNJXN3cUcEHun6rQw==" />


    <link rel="stylesheet" media="all" href="/assets/application.debug-ef0aee85ed1b20aaae7795cdebf5cd22b78468ca2f504f996aa706e6abdde343.css" data-turbolinks-track="reload" />
    <script src="/packs/js/application-b2160c96b9ff08432dd6.js" data-turbolinks-track="reload"></script>
  </head>

  <body>
    <h1>New Test</h1>


    <form class="simple_form new_test" id="new_test" novalidate="novalidate" data-client-side-validations="{&quot;html_settings&quot;:{&quot;type&quot;:&quot;SimpleForm::FormBuilder&quot;,&quot;error_class&quot;:null,&quot;error_tag&quot;:&quot;div&quot;,&quot;wrapper_error_class&quot;:&quot;field_with_errors&quot;,&quot;wrapper_tag&quot;:&quot;div&quot;,&quot;wrapper_class&quot;:&quot;input&quot;,&quot;wrapper&quot;:&quot;default&quot;},&quot;number_format&quot;:{&quot;separator&quot;:&quot;.&quot;,&quot;delimiter&quot;:&quot;,&quot;},&quot;validators&quot;:{&quot;test[name]&quot;:{&quot;presence&quot;:[{&quot;message&quot;:&quot;can&#39;t be blank&quot;}],&quot;length&quot;:[{&quot;messages&quot;:{&quot;minimum&quot;:&quot;is too short (minimum is 5 characters)&quot;},&quot;minimum&quot;:5}]}}}" action="/tests" accept-charset="UTF-8" data-remote="true" method="post">



      <div class="form-inputs">
        <div class="input string required test_name"><label class="string required" for="test_name"><abbr title="required">*</abbr> Name</label><input class="string required" type="text" name="test[name]" id="test_name" /></div>
      </div>

      <div class="form-actions">
        <input type="submit" name="commit" value="Create Test" class="btn" data-disable-with="Create Test" />
      </div>
    </form>

    <a href="/tests">Back</a>

  </body>
</html>
```
### Browser's development console output*
check Actual behavior*
### Additional JavaScript Libraries*
_If your issue depends on other JavaScript libraries, please list them here. E.g: *Bootstrap Modal v3.3.7, jQuery UI Datepicker 1.12.4*._
I don't know. I just use ```yarn add jquery```.
### Repository demostrating the issue
Debugging CSV issues is a time consuming task. If you want to speed up things, please
provide a link to a repository showing the issue.

https://github.com/no1-knows/BUGcheck_client_side_validation_with_simple_form

Sorry for no histroy.
What I did is setting jquery, simple_form, csv, csv_simple_form.
Make test scaffold name:string
and Steps to reproduce*

---

**\*** Failure to include this requirement may result in the issue being closed.

**Comment by tagliala** [comment]
Hi,

thanks for reporting this and thanks for the reproducible test case, it really helped.

So, it appears that when `error` is commented out in favor od `full_error`, this plugin does not detect the error wrapper class and crashes.

I will fix this as well as another issue

**Comment by tagliala** [comment]
@no1-knows released 10.0.0

I've bumped the major version number but in most of the use cases there should be not breaking changes
